src/dipdup/datasources/tzkt/datasource.py

```diff
@@ -811,8 +810,8 @@ def _get_ws_client(self) -> SignalRClient:
         self._logger.info('Creating websocket client')
         self._ws_client = SignalRClient(
             url=f'{self._http._url}/v1/events',
-            # NOTE: 1 MB default is not enough for big blocks
-            max_size=DEFAULT_MAX_SIZE * 10,
+            # NOTE: It's safe. Remove comment after updating pysignalr.
+            max_size=None,  # type: ignore
```

Comment:
DEFAULT_MAX_SIZE is 1Mb https://github.com/baking-bad/pysignalr/blob/bfef494d240852b8e8b2cc974f1766339394a7c8/src/pysignalr/transport/websocket.py#L31
before: 10Mb -> after: 1Mb ?

Reply:
Explicitly setting `max_size=None` disables the message size limit. Currently `pysignalr` passes this argument to the `websockets.connect` method as is, where the argument is `Optional[int]`.